### PR TITLE
Require Python 2.4 compatibile packages

### PR DESCRIPTION
**Pull request by ale-rt**
This patch saves your life if you want to use this package in a Plone 3 site

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,10 +14,14 @@
...
-except ImportError:
-    requirements.append('simplejson')
+if version_info[0] == 2 and version_info[1] < 6:
+    # If this happens you are probably using Python 2.4
+    # According to the documentation of simplejson, since version 2.1.0,
+    # Python 2.4 is no longer supported but may still work
+    # This line will probably pull in simplejson 2.3.3
+    requirements.append('simplejson < 2.4')
+    # To my knowledge 1.3 work fine
+    requirements.append('plone.app.transmogrifier < 1.4')
```
Comment (mauritsvanrees) [inline setup.py]: This means you get `plone.app.transmogrifier` twice in the requirements, once with and once without version restriction. Does this work? Might be better to remove it from the first list, and add it with an `else` in this condition.

Other than that: +1.

Comment (ale-rt) [inline setup.py]: I found it surprisingly working!
```
(testenv2.4)ale@emily:~/Code/testenv2.4/collective.jsonmigrator$ ../bin/python2.4 setup.py install 2> /dev/null |grep plone.app.transmogrifier
Searching for plone.app.transmogrifier<1.4
Reading http://pypi.python.org/simple/plone.app.transmogrifier/
Best match: plone.app.transmogrifier 1.3
```
But I see couple of glitches now:

1. there are some `except Exception as e` which are not python2.4 compatible
2. the produced `requires.txt` is not aware of this version limitation

I will try to fix both.

**Comment by wesleybl** [comment]
Python 2.7+ is now required.

**Comment by ale-rt** [comment]
I just restored the branch... you never know if somebody wants to use this package to migrate a very old site.

**Comment by wesleybl** [comment]
> I just restored the branch... you never know if somebody wants to use this package to migrate a very old site.

Sorry.

**Comment by ale-rt** [comment]
No problem at all, you are doing an amazing job fixing all this things!
Please go on and thank you!

**Pull request closed without merging**